.github/workflows/pr-auth-dependabot.yml

```diff
@@ -0,0 +1,26 @@
+name: auth dependabot myself
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  dependabot-comments:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14.15.4"
+      - run: yarn install
```

Comment:
Follow up to [this thread](https://github.com/marilyn79218/react-lazy-show/pull/44/files#r663794099), but don't forget to install dependencies before ahead.